Index: ql/src/java/org/apache/hadoop/hive/ql/exec/persistence/HashMapWrapper.java
===================================================================
--- ql/src/java/org/apache/hadoop/hive/ql/exec/persistence/HashMapWrapper.java	(revision 1028145)
+++ ql/src/java/org/apache/hadoop/hive/ql/exec/persistence/HashMapWrapper.java	(working copy)
@@ -325,7 +325,6 @@
       recman = RecordManagerFactory.createRecordManager(jdbmfile, props);
       long recid = recman.getNamedObject( "hashtable" );
       if ( recid != 0 ) {
-          System.out.println( "Reloading existing hashtable..." );
           pHash = HTree.load( recman, recid );
       }else{
         LOG.warn("initiliaze the hash table by jdbm file Error!");
Index: ql/src/java/org/apache/hadoop/hive/ql/exec/MapJoinOperator.java
===================================================================
--- ql/src/java/org/apache/hadoop/hive/ql/exec/MapJoinOperator.java	(revision 1028145)
+++ ql/src/java/org/apache/hadoop/hive/ql/exec/MapJoinOperator.java	(working copy)
@@ -165,7 +165,6 @@
 
     String currentInputFile = HiveConf.getVar(hconf,
         HiveConf.ConfVars.HADOOPMAPFILENAME);
-    LOG.info("******* Load from JDBM File: input : "+ currentInputFile);
 
     String currentFileName;
 
@@ -178,7 +177,6 @@
     try{
       if(localMode){
         //load the jdbm file from tmp dir
-        LOG.info("******* Load from tmp file uri ***");
         tmpURI= this.getExecContext().getLocalWork().getTmpFileURI();
         for(Map.Entry<Byte, HashMapWrapper<MapJoinObjectKey, MapJoinObjectValue>> entry: mapJoinTables.entrySet()){
           pos = entry.getKey();
@@ -193,7 +191,6 @@
         }
       }else{
         //load the jdbm file from distributed cache
-        LOG.info("******* Load from distributed Cache ***:");
          Path[] localFiles= DistributedCache.getLocalCacheFiles(this.hconf);
          for(int i = 0;i<localFiles.length; i++){
            Path path = localFiles[i];
@@ -225,7 +222,6 @@
              throw new HiveException();
            }
          }
-         LOG.info("******* End of loading *******:");
 
       }
     }catch (Exception e){
Index: ql/src/java/org/apache/hadoop/hive/ql/exec/JDBMSinkOperator.java
===================================================================
--- ql/src/java/org/apache/hadoop/hive/ql/exec/JDBMSinkOperator.java	(revision 1028145)
+++ ql/src/java/org/apache/hadoop/hive/ql/exec/JDBMSinkOperator.java	(working copy)
@@ -429,7 +429,6 @@
           FileSystem fs = tmpURIPath.getFileSystem(this.getExecContext().getJc());
           fs.copyFromLocalFile(localPath, tmpURIPath);
           LOG.info("Upload 1 JDBM File to: "+tmpURIPath);
-          System.out.println("Upload 1 JDBM File to: "+tmpURIPath);
           //remove the original jdbm tmp file
           hashTable.close();
         }
